feature/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go

```diff
@@ -24,11 +24,76 @@ import (
 	"github.com/openconfig/ondatra"
 	"github.com/openconfig/ondatra/gnmi"
 	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+type Testcase struct {
+	name string
+	fn   func(t *testing.T)
+}
+
+var (
+	QoSConfigTestcases = []Testcase{
+		{
+			name: "testQoSForwadingGroupsConfig",
+			fn:   testQoSForwadingGroupsConfig,
+		},
+		{
+			name: "testQoSClassifierConfig",
+			fn:   testQoSClassifierConfig,
+		},
+		{
+			name: "testQoSInputIntfClassifierConfig",
+			fn:   testQoSInputIntfClassifierConfig,
+		},
+		{
+			name: "testECNConfig",
+			fn:   testECNConfig,
+		},
+		{
+			name: "testSchedulerPoliciesConfig",
+			fn:   testSchedulerPoliciesConfig,
+		},
+		{
+			name: "testQoSOutputIntfConfig",
+			fn:   testQoSOutputIntfConfig,
+		},
+	}
+)
+var (
+	QoSConfigQosTestcases = []Testcase{
+
+		{
+			name: "testQoSCiscoClassifierConfig",
+			fn:   testQoSCiscoClassifierConfig,
+		},
+		{
+			name: "testCiscoSchedulerPoliciesConfig",
+			fn:   testCiscoSchedulerPoliciesConfig,
+		},
+	}
 )
 
 func TestMain(m *testing.M) {
 	fptest.RunTests(m)
 }
+func TestQosConfigTests(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	switch dut.Vendor() {
+	case ondatra.CISCO:
+		for _, tt := range QoSConfigQosTestcases {
```

Comment:
Looks to me this PRs skip several test categories for Cisco: FowardingGroup, InputIntfClassifer, ECN, OutputIntf?

Reply:
All the configs are there in the tests. It is combined into two testcases of classifier and scheduler.
The  reason being , in the original tests , we are reinitializing qos in each tests.
For eg:- TestQoSInputIntfClassifierConfig  we are using a replace at qos to apply classifier configs at interface.
But since in this test , since we reinitialize the qos , the classifier configs that was configured in earlier test is lost